assets/js/experience.js

```diff
@@ -57,60 +116,50 @@ const showCards = () => {
 document.addEventListener("DOMContentLoaded", showCards);
 
 
-//  Work experience cards
+// Mentorship Card
 
 
-const experiencecards = document.querySelector(".experience-cards");
-const exp = [
-  {
-    title: "Reasearch and Development Intern",
-    cardImage: "assets/images/experience-page/UCL.JPG",
-    place: "UCL",
-    time: "(May2020-present)",
-    desp: "<li>Automatic detection of Orthopedic Implants using Deep Learning.</li> <li>Collection of radiographs, data cleaning and database management.</li> <li>Image Quality Analysis for Bio-Medical Images.</li> <li>Serving Models using TensorFlow JS.</li> <li>Experimenting with Few Shot Learning, Adversarial Training and Class Activation Maps.</li>",
-  },
+const mentorshipcards = document.querySelector(".mentorship-cards");
+const mentor = [
   {
-    title: "Robotics and Embedded system Intern",
-    cardImage: "assets/images/experience-page/lema-labs.png",
-    place: "LEMA LABS",
-    time: "(Jun-Jul,2019)",
-    desp: "<li>Design low cost prototyping boards for teaching robotics to underprivileged children.</li> <li>Worked on integration of Face Recognition based models with Raspberry Pi and Intel Movidius.</li> <li>Image Quality Analysis for Bio-Medical Images.Experimented with various algorithms for Lane Detection as well as Shortest Paths for Autonomous Vehicles.</li>",
+    title: "HakinCode",
+    image: "assets/images/experience-page/hakin.png",
+    time: "06/2020 - 08/2020",
+    desp1: "Mentored for a project named Awesome Portfolio Project. It helps to instantly create portfolio.",
```

Comment:
Description can be stored in a single variable where one can give points in an ordered/ unordered list

Reply:
okay 👍